bot.py

```diff
@@ -170,10 +170,20 @@ async def set_leaderboard_channel(ctx, channel: discord.TextChannel):
     main_leaderboard = Leaderboard(db_handler=db_handler, channel=channel)
     await ctx.respond("Leaderboard channel updated.")
 
-# Set Soloque Leaderboard Channel
 @commands.has_role("Staff")
 @bot.slash_command(description="Staff only command. Sets the leaderboard output channel.")
-async def set_soloque_channel(ctx, channel: discord.TextChannel):
+async def refresh_soloque_channel(ctx: discord.ApplicationContext):
+    global solo_queue_leaderboard
```

Comment:
nit: doesn't have to be this MR but we should move this and the SP leaderboard to the `global_objects.py` files at some point.